src/AWSS3.jl

```diff
@@ -819,7 +836,9 @@ function s3_list_keys(aws::AbstractAWSConfig, bucket, path_prefix=""; kwargs...)
     return (o["Key"] for o in s3_list_objects(aws, bucket, path_prefix; kwargs...))
 end
 
-s3_list_keys(a...; b...) = s3_list_keys(global_aws_config(), a...; b...)
+function s3_list_keys(bucket, path_prefix=""; kwargs...)
+    return s3_list_keys(global_aws_config(), bucket, path_prefix; kwargs...)
+end
```

Comment:
Avoid redefining default values:
```suggestion
s3_list_keys(args...; kwargs...) = s3_list_keys(global_aws_config(), args...; kwargs...)
```